Repositories.Tests/ProjectRepositoryTest.cs

```diff
@@ -75,9 +76,54 @@ public async Task GetAllWithUsersAsyncTest_NoProjects([UserDataSource(100)]List<
         }
 
         /// <summary>
-        /// Check if the search is returning the correct projects based on the search terms
+        /// Gets all with users asynchronous test is public email false.
         /// </summary>
-        /// <param name="projects">The projects which are used as data to test</param>
+        /// <param name="project">The project.</param>
+        /// <param name="user">The user.</param>
+        [Test]
+        public async Task GetAllWithUsersAsyncTest_IsPublicEmail_False(
+            [ProjectDataSource]Project project, [UserDataSource]User user)
+        {
+            user.IsPublic = false;
+            DbContext.Add(user);
+            project.User = user;
+            // Seed database
+            DbContext.Add(project);
+            await DbContext.SaveChangesAsync();
+
+            // Test
+            List<Project> retrieved = await Repository.GetAllWithUsersAsync();
+            Assert.AreEqual(1, retrieved.Count);
+            Assert.AreEqual(retrieved[0].User.Email, Defaults.Privacy.RedactedEmail);
+        }
+
+        /// <summary>
+        /// Gets all with users asynchronous test is public email false.
+        /// </summary>
+        /// <param name="project">The project.</param>
```

Comment:
The documentation in this class (mostly the parameter documentation) is very vague. Please add a proper description of what the parameters do.